frontend/src/components/NextPhaseButton/NextPhaseButton.tsx

```diff
@@ -0,0 +1,40 @@
+import rootStore from '../../stores/RootStore';
+import Button from '../Button/Button';
+import { observer } from 'mobx-react';
+
+let buttonLabel = 'Next phase';
+
+export enum GamePhases {
+  TILE_PLACEMENT = 1,
+  MEEPLE_PLACEMENT,
+  SCORE_PHASE,
```

Comment:
let's stick with phase name instead of number
so it will be TILE_PLACEMENT = "Tile placement phase" etc.